src/components/SlidePanel/_slide-panel.scss

```diff
@@ -0,0 +1,15 @@
+.slide-panel {
+  background-color: white;
+  box-shadow: 0 1px 5px 1px rgba(51, 51, 51, 0.2);
+  height: calc(100% - 25px);
+  position: absolute;
+  right: 0;
+  top: 40px; // height of the top bar
```

Comment:
If your going to start 40px down you need to minus that from the height. Therefore the height value above should be `calc(100% - 40px)`. As they are the same you could use a variable to keep them aligned and easier read. Removing the need for the comment.

Reply:
Done